scripts/phase8/llm_background_load.py

```
@@ -0,0 +1,133 @@
+        "notification_email": f"phase8-llm-{tenant_id.hex[:8]}@example.invalid",
+        "shopify_webhook_secret": "phase8-llm-shopify-secret",
+        "stripe_webhook_secret": "phase8-llm-stripe-secret",
+        "paypal_webhook_secret": "phase8-llm-paypal-secret",
+        "woocommerce_webhook_secret": "phase8-llm-woo-secret",
+        "created_at": now,
+        "updated_at": now,
+    }
+    insert_cols = [col for col in required if col in payload]
+    missing = sorted(col for col in required if col not in payload)
+    if missing:
+        raise RuntimeError(f"Missing required tenant columns: {', '.join(missing)}")
```

Ruff (0.15.0)

[warning] Avoid specifying long messages outside the exception class

(TRY003)